docs/data/layer_types.md

```diff
@@ -4,38 +4,70 @@ sidebar_position: 2
 
 # Layer Types
 
-## Data types
+## Internal Layers 
 
-GOAT supports various geometry types for spatial data. When uploading your data files, you can include the following types of geospatial data:
+In the spatial data architecture of the GOAT system, **Internal Layers** refer to key components that are stored and managed internally. These include the [Feature Layer](#1-feature-layer/), which contains spatially referenced geographic features that are fundamental to mapping and spatial analysis tasks. Alongside this is the [Table Layer](#2-table-layer), which consists of non-spatial, tabular data. This data is essential for carrying out detailed attribute analysis and maintains a link to the spatial features, although it does not itself contain spatial references.
 
-- **Point:** For datasets consisting mainly of points. Supported geometry types are Point and Multi-Point.
-- **Line:** For datasets that represent linear features. Supported geometry types are Line String and Multi-Line String.
-- **Polygon:** For datasets that define area-based features. Supported geometry types are Polygon and Multi-Polygon.
-- **No geometry:** For uploading non-spatial data.
+### Data Types
+
+In the GOAT system approach to data management, there are specific steps for integrating and fusing different data sets. These steps are tailored to the local context. In addition, further data processing steps are taken to ensure accuracy, completeness, and reliability.
```

Comment:
Do we need this here? I am seeing this content more part of the data catalogue. Here it is just about the functionality of the tool and the data types.